glide.go

```diff
@@ -87,6 +88,11 @@ func main() {
 			Usage:  "The temp directory to use. Defaults to systems temp",
 			EnvVar: "GLIDE_TMP",
 		},
+		cli.UintFlag{
```

Comment:
Please name the parameter something that describes its purpose. `http-timeout` would be fine.